wrapper/src/main/java/com/splunk/support/lambda/configuration/JaegerThriftSpanExporterFactory.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Splunk Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.splunk.support.lambda.configuration;
+
+import com.google.auto.service.AutoService;
+import io.jaegertracing.thrift.internal.senders.HttpSender;
+import io.opentelemetry.exporter.jaeger.thrift.JaegerThriftSpanExporter;
+import io.opentelemetry.exporter.jaeger.thrift.JaegerThriftSpanExporterBuilder;
+import io.opentelemetry.sdk.autoconfigure.ConfigProperties;
+import io.opentelemetry.sdk.autoconfigure.spi.ConfigurableSpanExporterProvider;
+import io.opentelemetry.sdk.trace.export.SpanExporter;
+import okhttp3.OkHttpClient;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@AutoService(ConfigurableSpanExporterProvider.class)
+public class JaegerThriftSpanExporterFactory implements ConfigurableSpanExporterProvider {
+  private static final Logger log = LoggerFactory.getLogger(JaegerThriftSpanExporterFactory.class);
+
+  static final String SIGNALFX_AUTH_TOKEN = "signalfx.auth.token";
```

Comment:
Just FYI for the future: we're planning to rename that to `SPLK_ACCESS_TOKEN` (:sob: ) or `SPLUNK_ACCESS_TOKEN` in splunk-otel-java to match all our other distros, it'd be quite good if the lambda also follows this convention

Reply:
Yep, will follow (assuming will rememmber about that :P )